frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -337,6 +344,7 @@ export const experimentLogic = kea<experimentLogicType>([
                             ...values.experiment?.parameters,
                             recommended_running_time: runningTime,
                             recommended_sample_size: sampleSize,
+                            minimum_detectable_effect: values.minimumDetectableChange,
```

Comment:
Assuming you'll handle 'change experiment goal' in a follow up?

Reply:
also, I'm a bit confused how this is working currently, since new UI doesn't really set running time and sample size?

Reply:
(without the above^ the goal doesn't change no matter what the MDE is)

Reply:
Ah indeed, I forgot to transfer that part from the old UI. This one, right? https://github.com/PostHog/posthog/blob/data-collection-mde/frontend/src/scenes/experiments/Experiment.tsx#L134

Because we're not setting it, it defaults to `{ exposure: 0, sampleSize: 0 }`

Reply:
correct! The old code is hacky though, given you have trend and funnel results with you in the logic, can clean this up for new UI too 👍